.gitmodules

```diff
@@ -6,7 +6,7 @@
 	url = https://github.com/percona/Percona-TokuBackup.git
 [submodule "storage/rocksdb/rocksdb"]
 	path = storage/rocksdb/rocksdb
-	url = https://github.com/facebook/rocksdb.git
+	url = https://github.com/kamil-holubicki/rocksdb.git
```

Comment:
Once our work is finalized, it will be submitted to Rocsdb and may take up to a year to get accepted, then incorporated into Facebook MySQL, then merged to Percona Server.  In the meanwhile, we will have no choice but to maintain a Percona fork of RocksDB.  We have had to do this once or twice in the past in order to keep a single Percona commit on top of the appropriate point-in-time that matches what we have merged in from the Facebook MySQL/MyRocks<-->RocksDB.